src/test/scala/com/cognite/sdk/scala/v1/ThreeDTest.scala

```diff
@@ -55,6 +55,15 @@ class ThreeDTest extends SdkTest with ReadBehaviours with WritableBehaviors {
       false
     )
   )
+  it should "support retrieving an item by id" in {
```

Comment:
If we implement `RetrieveByIds` we can do `it should behave like readableWithRetrieve(client.threeDRevisions(4222532244684431L), idsThatDoNotExist, supportsMissingAndThrown = false)`
And we should change `readableWithRetrieve` to make use of `retrieveById` at least once.